contexts/vc-revocation-list-v1.jsonld

```diff
@@ -0,0 +1,32 @@
+{
+  '@context': {
```

Comment:
I noted that the version 1.1 stuff seen elsewhere is not here.  Is that OK?

Reply:
Yeah, I'm not sure. That's something I need to check on. Some changes to JSON-LD 1.1 made the version 1.1 stuff optional and the thing I need to check on is whether or not that causes any problems with `@protected` term redefinitions or not. In other words, if you don't include `@version: 1.1` in a term definition, and someone else defines the term in the same way except they do include `@version: 1.1`, does the processor blow up or allow it as the definition should be identical (ideally this)? @davidlehn -- do you know off hand?

Reply:
Not sure offhand.  Practically speaking, if the explicit and implicit versions are compatible, I'd think it would be ok.  I'm not sure what kind of test coverage we have for the various permutations of that.  And not sure what happens when a future 1.2 exists and the version combinations that could arise.